conf/nginx.conf

```
@@ -1,11 +1,13 @@
 #sub_path_only rewrite ^__PATH__$ __PATH__/ permanent;
 location __PATH__/ {
 
-    proxy_pass         http://127.0.0.1:__PORT__/;
+    proxy_pass         http://127.0.0.1:__PORT__/__PATH2__;
```

I think we will end up by having two slash between port and path...
```suggestion
    proxy_pass         http://127.0.0.1:__PORT____PATH2__;
```

---

No because of the definition of `$path2` cf

https://github.com/YunoHost-Apps/xwiki_ynh/blob/466724f40bc18fc3ff2f6c0c071f05de657ba368/scripts/_common.sh#L25